### DISCUSSION TITLE
BTS styled entry for clear style

### DISCUSSION THREAD

**Discussion by ItzAlmaZ** [BEEmod/BEE2-items#4139] [Feature Suggestions]
1. you appear in BTS and enter the test chamber through the observation hall
2. You appear in the pipe and arrive at the test chamber and a hatch on a non-portal surface closes behind you
3. you enter the test chamber through an open panel in the wall that will close after entering the chamber
Everything is done ugly, give an answer if the spawn without an elevator is not implemented without a hammer
(1) ![sp_a2_column_blocker0000](https://user-images.githubusercontent.com/104623098/166451701-c3d136fb-aa63-4ef4-91a5-bbd532521e58.jpg)

https://user-images.githubusercontent.com/104623098/166451856-46425355-f87b-4e90-b996-960db42583de.mp4

https://user-images.githubusercontent.com/104623098/166451866-f00e9492-4ade-44b1-8dc5-60feb2c15bd6.mp4

**Comment by SimpleFoxOfficial** [comment]
first image is more likely for a combined style with overgrown and clean. Window up the entry is cannot be realized because you can use only one block for entry.

Edit: sorry, didn't read the text. How then you will add the classic entry door for decoration?

**Comment by vrad-exe** [comment]
We're not adding this, the only thing it'd be used for is story maps

**Reply by ItzAlmaZ** [reply]
Yes, I have thought about it, these entrances are inappropriate and stupid. But what about what happens in chapter 9 as soon as we get out of the old aperture. We enter the cell through a broken panel in the wall. And then we go through the chamber as usual
